### PR TITLE
feat(aws): More ApplicationAutoScaling resources

### PR DESCRIPTION
**Pull request by disq**
Some of #[7921](https://github.com/cloudquery/cloudquery/issues/7921)

### CODE REVIEW

File: plugins/source/aws/resources/services/applicationautoscaling/scalable_targets.go [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+)
+
+func ScalableTargets() *schema.Table {
+	tableName := "aws_applicationautoscaling_scalable_targets"
+	return &schema.Table{
+		Name:        tableName,
+		Description: `https://docs.aws.amazon.com/autoscaling/application/APIReference/API_ScalableTarget.html`,
+		Resolver:    fetchScalableTargets,
+		Multiplex:   client.ServiceAccountRegionNamespaceMultiplexer(tableName, "application-autoscaling"),
+		Transform:   transformers.TransformWithStruct(&types.ScalableTarget{}, transformers.WithPrimaryKeys("ResourceId")),
```
Comment (disq) [inline plugins/source/aws/resources/services/applicationautoscaling/scalable_targets.go:21]: `ResourceId` can be bunch of things, some ARN, some not. Refer to the URL in the description for details.

File: website/tables/aws/aws_applicationautoscaling_scheduled_actions.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+# Table: aws_applicationautoscaling_scheduled_actions
+
+This table shows data for Applicationautoscaling Scheduled Actions.
```
Comment (hermanschaaf) [inline website/tables/aws/aws_applicationautoscaling_scheduled_actions.md:3]: Please add another exception for `Applicationautoscaling` to plugin.go for the docs generation 😄

File: plugins/source/aws/resources/services/applicationautoscaling/scheduled_actions_mock_test.go [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+	m.EXPECT().DescribeScheduledActions(gomock.Any(), gomock.Any(), gomock.Any()).Return(
+		output,
+		nil,
+	)
+
+	return services
+}
+
+func TestScheduledActions(t *testing.T) {
+	client.AllNamespaces = []string{"test-namespace"} // Just one
```
Comment (hermanschaaf) [inline plugins/source/aws/resources/services/applicationautoscaling/scheduled_actions_mock_test.go:35]: Is there any way we could avoid changing this? Maybe it should be a property on the client instead.

As long as `AllNamespaces` is only used by this package and all the tests here set it to the same value, it should be okay, but as soon as one of those assumptions change we could have a fairly hard-to-find bug in our testing.

I see it was like this in the code before already, so consider this comment non-blocking.

File: website/tables/aws/aws_applicationautoscaling_scheduled_actions.md (hunk 2 above)
Comment (disq) [reply website/tables/aws/aws_applicationautoscaling_scheduled_actions.md:3]: oof it reindented everything so I added it to the very top

**Review by hermanschaaf — APPROVED** [review]

**Pull request merged**